_build/redirection_map

```diff
@@ -458,6 +458,7 @@
 /templating/embedding_controllers /templates#embedding-controllers
 /templating/inheritance /templates#template-inheritance-and-layouts
 /testing/doctrine /testing/database
+/translation/templates /translation
```

Comment:
```suggestion
/translation/templates /translation#translation-in-templates
```

We recently made the redirection tool anchor aware :partying_face:

Reply:
Thank you